RELEASE.md

```diff
@@ -16,6 +16,7 @@
   * New navigation bar in Curses-based UI
   * NodeStepper (command `invoke_stepper`) now uses intermediate tensor dumps. It also uses `TensorHandles` as direct feeds during successive `cont` calls for improved performance and reduced memory consumption.
 * Initial release of installation guides for Java, C, and Go.
+* Added Text Dashboard to TensorBoard.
```

Comment:
nit: Is it usually called "Text Summary", instead of "Text Dashboard? (I have personally never heard of the latter term).

cc @dandelionmane @dsmilkov